apps/doc-scanner/README.md

```diff
@@ -0,0 +1,51 @@
+# GEMI Document Scanner
+
+A test Node.js tool for processing Greek company documents (GEMI) to extract metadata and create historical timelines.
+
+## Setup
+
+1. Install dependencies:
+   Navigate to the project directory `doc-scanner-app`
+
+```sh
+npm install
+```
+
+2. Create a .env file in the project root:
+
+```sh
+GEMINI_API_KEY=your_api_key_here
+```
+
+3. Create input folder under src/data:
+
+```
+input/
+  {GEMI_ID}/     # Place source documents here
+    doc1.pdf
+    doc2.docx
+output/          # Output folder (created automatically)
+```
+
+## Usage
+
+1. Place your PDF/DOCX documents in a folder named with the GEMI ID under data_src
+2. Run the processor:
```

Comment:
These 2 blocks indicate different file structure, can you clarify which one is the right one and update the readme?